.cirrus.yml

```diff
@@ -9,3 +11,5 @@ Linux_task:
   clone runtime_script: git clone https://github.com/dotnet/runtime --branch master --single-branch --depth 1
 
   build runtime_script: runtime/build.sh -c release -cross -os FreeBSD -ci
+
+  buildTests  runtime_script: runtime/build.sh -c release -cross -os FreeBSD -ci -subset Libs.Tests /p:archivetests=true
```

Comment:
Would it make it slower (i.e. build some bits again), if we put it under a separate linux task? I was thinking:

```diff
-  Linux_task:
+ Linux build_task:
..
   build runtime_script: runtime/build.sh -c release -cross -os FreeBSD -ci
..

+ Linux test build_task:
..
+   buildTests runtime_script: runtime/build.sh -c release -cross -os FreeBSD -ci -subset Libs.Tests /p:archivetests=true

```

Reply:
we can try it to get the data. I don't know the implication is of doing so. Would that get the tree as the previous task left it or would it it start from fresh tree?

Reply:
It would be a fresh tree. Each task is a separate instance, tasks can depend on each other or run in parallel, tasks can share data via local HTTP cache server. It will save some time if we don't need to zip the the whole `artifacts` directory for the next task.

Reply:
ok. I'll give it some try to get data. It seems like it is failing as it is trying to (incorrectly) use 
```
 /tmp/cirrus-ci-build/runtime/artifacts/bin/testhost/netcoreapp5.0-FreeBSD-Release-x64/shared/Microsoft.NETCore.App/5.0.0/libcoreclr.so, error : libthr.so.3: cannot open shared object file: No such file or directory 
```

Reply:
> /tmp/cirrus-ci-build/runtime/src/libraries/Microsoft.XmlSerializer.Generator/tests/Microsoft.XmlSerializer.Generator.Tests.csproj(44,5): error MSB3073: The command "export DOTNET_MULTILEVEL_LOOKUP=0 && "/tmp/cirrus-ci-build/runtime/artifacts/bin/testhost/netcoreapp5.0-FreeBSD-Release-x64/dotnet" --fx-version 5.0.0 /tmp/cirrus-ci-build/runtime/artifacts/bin/Microsoft.XmlSerializer.Generator.Tests/netcoreapp5.0-Release/dotnet-Microsoft.XmlSerializer.Generator.dll /tmp/cirrus-ci-build/runtime/artifacts/bin/Microsoft.XmlSerializer.Generator.Tests/netcoreapp5.0-Release/Microsoft.XmlSerializer.Generator.Tests.dll --force --quiet" exited with code 137.

Is this an indication that it should not trying to run FreeBSD binary (`"/tmp/cirrus-ci-build/runtime/artifacts/bin/testhost/netcoreapp5.0-FreeBSD-Release-x64/dotnet"`) on Linux box?